participants/notify_irc.py

```diff
@@ -10,7 +10,10 @@ def notify(self, msg=["No 'message' for notify_irc"], channel="#meego-boss", hig
         ircbot.connect((self.ircbot_host, self.ircbot_port))
         for item in msg:
             # split possible lines
```

Comment:
add comment explaining that some bots handle multi-line messages, irc bot doesn't